.travis.yml

```diff
@@ -1,87 +1,50 @@
-language: python
-
-python:
-    - 3.7
+language: c
 
 # Setting sudo to false opts in to Travis-CI container-based builds.
 sudo: false
 
 
 env:
     global:
-        # The following versions are the 'default' for tests, unless
-        # overridden underneath. They are defined here in order to save having
-        # to repeat them for all configurations.
-        - NUMPY_VERSION=stable
-        - ASTROPY_VERSION=stable
-        - SETUP_CMD='test'
-        - PIP_DEPENDENCIES=''
-        - CONDA_DEPENDENCIES='scipy matplotlib'
+        - TOXENV=''
+        - TOXARGS=''
+        - TOXPOSARGS=''
 
-        # For headless testing of code that imports PyPlot:
-        - MPLBACKEND=Agg
-    matrix:
-        # Make sure that egg_info works without dependencies
-        - SETUP_CMD='egg_info'
+        - TOX_CMD='tox --'
+        - TOX_ARGS=''
 
 matrix:
+
     include:
         # do the actual tests.
-        # Do a coverage test in Python 3.
-        - python: 3.7
-          env: SETUP_CMD='test --coverage'
+        # Do a test in Python 3.7
+        - env: TOXENV='coverage' TOX_ARGS=''
 
         # Check accelerated math version too
-        - python: 3.7
-          env: SETUP_CMD='test --coverage' PIP_DEPENDENCIES='numexpr'
```

Comment:
The existing version of this runs coverage tests on both the regular and with-numexpr versions. That results in showing higher coverage on the test report (which is a more accurate depiction of the actual test coverage, given that the numexpr test case exists). I assume codecov can probably also handle merging test reports from multiple test builds? 

(Not a priority for an immediate change, but perhaps you know enough of tox configs and codecov now to see how to add that in?)